build-docker.sh

```diff
@@ -19,17 +19,33 @@ cp -r ~/.m2/repository/io/karatelabs ${KARATE_REPO}
 mvn package -B -ntp -P fatjar -DskipTests -f karate-core/pom.xml
 cp karate-core/target/karate-${KARATE_VERSION}.jar karate-docker/karate-chrome/target/karate.jar
 
-# hack for apple silicon
-# export DOCKER_DEFAULT_PLATFORM=linux/amd64
+# setup multiplatform build
+docker buildx rm multiplatform-builder
+docker buildx create --name multiplatform-builder
+docker buildx use multiplatform-builder
 
 # build karate-chrome docker image that includes karate fatjar + maven jars for convenience
-docker build -t karate-chrome karate-docker/karate-chrome
+# Only possible for linux/amd64 as chrome not available on linux/arm64
+docker buildx build --platform linux/amd64 --cache-from=type=local,src=./target/docker --cache-to=type=local,dest=./target/docker -t karate-chrome -t karate-chrome:latest karate-docker/karate-chrome
+# build karate-chromium docker image that includes karate fatjar + maven jars for convenience
+# Both platform
+docker buildx build --platform linux/amd64,linux/arm64 --cache-from=type=local,src=./target/docker --cache-to=type=local,dest=./target/docker -t karate-chromium karate-docker/karate-chromium
```

Comment:
does the build scripts in the tests have the docker buildx plugin configured?

Reply:
nope but I can't check if it's by default in gitlab run. It's maybe the case as it's a common use case now

Reply:
nope but I can't check if it's by default in gitlab run. It's maybe the case as it's a common use case now